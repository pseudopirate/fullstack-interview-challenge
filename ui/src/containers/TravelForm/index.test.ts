import { DateTime } from "luxon";
import { FormValues, prepareFromValues } from ".";

describe('TravelForm', () => {
    describe('prepareFromValues(values: FormValues)', () => {
        const AUGUST_27_2022_MILLIS = 1661621605728;

        const values = {
            origin: 'TEST',
            destinations: ['TEST2'],
            durationOfStay: 5,
            departureDate: DateTime.fromMillis(AUGUST_27_2022_MILLIS),
        } as FormValues;

        it('should do smth', () => {
            expect(prepareFromValues(values)).toMatchSnapshot();
        });
    });
});