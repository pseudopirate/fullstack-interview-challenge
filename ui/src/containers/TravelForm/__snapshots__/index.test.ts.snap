// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TravelForm prepareFromValues(values: FormValues) should do smth 1`] = `
Object {
  "departureDate": 1661621605,
  "destinations": Array [
    "TEST2",
  ],
  "durationOfStay": 5,
  "origin": "TEST",
}
`;
